cme/modules/enum_av.py

```diff
@@ -3,106 +3,102 @@
 
 # All credit to @an0n_r0
 # project : https://github.com/tothi/serviceDetector
+# Module by @mpgn_x64
 
-from impacket.dcerpc.v5 import lsat, lsad
+from impacket.dcerpc.v5 import lsat, lsad, transport
 from impacket.dcerpc.v5.dtypes import NULL, MAXIMUM_ALLOWED, RPC_UNICODE_STRING
-from impacket.dcerpc.v5 import transport
 import pathlib
 
-
 class CMEModule:
-    """
-    Uses LsarLookupNames and NamedPipes to gather information on all endpoint protection solutions installed on the the remote host(s)
-    Module by @mpgn_x64
-    """
-
+    
     name = "enum_av"
-    description = "Gathers information on all endpoint protection solutions installed on the the remote host(s) via LsarLookupNames (no privilege needed)"
```

Comment:
AV/EDR is endpoint protection, why change the wording? also why remove the info about the method used?

Reply:
Got butchered when I was messing with the code. I typed it in manually.